lab/changelog.md

```diff
@@ -4,6 +4,31 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.4.0 - Dec 2, 2022
+
+## What's New
+
+* Support for manual transaction workflow has been enabled using transaction commands `BEGIN`, `COMMIT`, and `ROLLBACK`.
+* Cypher intellisense has been updated to suggest new Cypher features from Memgraph 2.4.0 such as:
+  * Privileges for user-role authorization.
+  * Commands and privileges for label-based authorization.
+  * Manual transaction commands: `BEGIN`, `COMMIT`, `ROLLBACK`.
+  * Checking configuration with `SHOW CONFIG`.
+  * All shortest path algorithm `allShortest`.
+  * Graph projection function `project`.
+  * Additional query module signature that accepts a projected graph as an optional first argument.
+* Graph results view will check for nodes and relationships in arrays and projected graphs. It simplifies
+  the visualization of a projected graph or an array of nodes/relationships without using `UNWIND`.
+
+## Bug Fixes
+
+* Once the table results view is selected, an automatic switch to the graph view won't happen on the next query run.
+* Exploring a dataset's query collection now works as expected. It opens up a query list to explore the dataset.
+* Missing detailed error message on a failed query in rich collections has been fixed.
```

Comment:
```suggestion
* Failed queries from the rich collections now return a detailed error message.
```

was this the point of this fix?

Reply:
Before, you would just get "Query failed" in the rich collection and that's it. Now it returns "what actually went wrong with the query" so you can fix it.